lib/hiera/backend.rb

```diff
@@ -8,6 +8,17 @@
 
 class Hiera
   module Backend
+    class LegacyBackendWrapper
```

Comment:
This code is Legacy tomorrow - better to use the version. This class adapts a Hiera 1x backend to the Hiera 2x API. So... Backend1xAdapter ?